tutorials/gas_compact_multiphysics/gold/openmc_thm_test_out.csv

```diff
@@ -0,0 +1,4 @@
+time,flux_integral,heat_source,k,k_std_dev,max_power,max_tally_err,min_power
```

Comment:
This file isn't used anywhere - it doesn't show up in any `tests`. Why is it here? There is already a test which is covering the `openmc_thm.i` case.

Reply:
My bad, I mistook the `Outputs/csv/file_base=openmc_thm_test_out` from `[openmc_thm_moose]`. I will rebase this commit out of the history, since all three issues were introduced by the same commit.